PythonDPPP/CMakeLists.txt

```diff
@@ -37,6 +37,9 @@ if(${Boost_PYTHON${BOOSTPY}_FOUND})
   
   find_package(Boost REQUIRED COMPONENTS python${BOOSTPY})
   include_directories(${Boost_INCLUDE_DIR})
+  # By default, Boost-Python has auto_ptr support, which triggers a compiler
+  # warning, since auto_ptr is deprecated -> Disable auto_ptr support.
+  add_definitions(-DBOOST_NO_AUTO_PTR)
```

Comment:
For what compiler / boost version is this necessary? auto_ptr was removed from c++14 I don't get any warnings with Boost, but Boost of course builds fine. If possible I'd rather keep Boost at its default settings, instead of adding extra options for old Boost versions.